content/2.0.html.haml

```diff
@@ -34,12 +34,11 @@ title: "Jenkins 2.0"
 
 
 %h1
-  Jenkins 2.0 preview
+  Jenkins 2.0 Overview
 
 %p
-  %strong
-    Jenkins 2.0
-  is an upcoming major version of Jenkins with a number of usability enhancements and new features.
+  Jenkins 2.0 is an automation engine with an unparalleled plugin ecosystem to support all of your favorite tools in your delivery pipelines, whether your goal is continuous integration, automated testing or continuous delivery. 
```

Comment:
It feels like an odd sound bite — the future tense should definitely be kept, since Jenkins 2.0 hasn't been released yet.  Jenkins 1.0 has the same features listed as those for Jenkins 2.0.  I don't think we've ever called Jenkins an "automation engine", and there are already soundbites on the [Press page](https://jenkins-ci.org/press/).